test-requirements.txt

```diff
@@ -1,6 +1,4 @@
---extra-index-url=https://wheelhouse.shopinvader.com/simple
-# TODO locomotivecms dep should be in the setup.py of the addon that needs it
-git+https://github.com/akretion/locomotivecms-python-client.git#egg=locomotivecms
+locomotivecms @ git+https://github.com/akretion/locomotivecms-python-client.git
```

Comment:
time to switch to https://github.com/shopinvader/locomotivecms-python-client ?

Reply:
done